docs/getting_started.ipynb

```diff
@@ -21,69 +21,62 @@
     "\n",
     "You will find your api key and secret at https://www.altmetric.com/explorer/settings.\n",
     "\n",
-    "You can use whatever you like to distribute secrets but, for the purposes of demonstrating how the API works, we are using [python-dotenv](https://pypi.org/project/python-dotenv/) and you must set it up with your own key and secret for the examples to work.\n",
+    "You can use whatever you like to your API key and secret but this notebook is designed to run in [Google Colab](https://colab.google/) and we are using the secrets store built into Colab to store the API key and secret securely.  To do this, open the \"Secrets\" by clicking the key icon on the left hand side of the page and make new secrets named `API_KEY` and `API_SECRET` and add the values from from you account settings in Explorer.\n",
```

Comment:
Is a verb missing here after "You can use whatever you like to..."?

Reply:
Yup.  Well spotted.  Fixed now